runsc/container/multi_container_test.go

```diff
@@ -156,6 +165,9 @@ func TestMultiContainerSanity(t *testing.T) {
 		if err := waitForProcessList(containers[0], expectedPL); err != nil {
 			t.Errorf("failed to wait for sleep to start: %v", err)
 		}
+		if err := waitForProcessList(containers[2], expectedPL); err != nil {
```

Comment:
IIUC, this would have passed even if PID namespace isn't working, because it would have found PID=1 from another container. I'd recommend adding another test for this specific case and checking the entire process list that is returned for the container. Please also add a test for PID namespace with a path to ensure the matching logic is working.

Reply:
Thanks for the suggestions!
 I have added the test.